### ISSUE TITLE
The tests break in master

### ISSUE THREAD

**Issue by peterwvj** [openfoodfacts/openfoodfacts-dart#42]
Three tests are currently failing in the master branch. I'm running the test from this commit fa0950b.

To produce the test errors run
```
flutter test test/
```
Relevant output:
```
TEST-MODE
00:03 +5 -1: /home/peter/git/openfoodfacts-dart/test/api_getProductRaw_test.dart: OpenFoodAPIClient get raw products get product test 1 [E]                                    
  Expected: 'Maisstärke'
    Actual: 'corn starch'
     Which: is different.
            Expected: Maisstärke ...
              Actual: corn starc ...
                      ^
             Differ at offset 0
  
  package:test_api                                   expect
  package:flutter_test/src/widget_tester.dart 348:3  expect
  test/api_getProductRaw_test.dart 35:7              main.<fn>.<fn>
...
0:04 +12 -2: /home/peter/git/openfoodfacts-dart/test/api_getProduct_test.dart: OpenFoodAPIClient get products get product Coca Cola Light [E]                                 
  Expected: <0.02>
    Actual: <0.01>
  
  package:test_api                                   expect
  package:flutter_test/src/widget_tester.dart 348:3  expect
  test/api_getProduct_test.dart 74:7                 main.<fn>.<fn>
  
00:04 +12 -2: /home/peter/git/openfoodfacts-dart/test/api_getProduct_test.dart: OpenFoodAPIClient get products get product tiny twists - Rold Gold Pretzels - 16 OZ. (1 LB) 453.6g

....
00:31 +29 -3: /home/peter/git/openfoodfacts-dart/test/api_getRobotoff_test.dart: OpenFoodAPIClient get robotoff ingredient spelling corrections get farine de blé spelling corrections [E]
  FormatException: Unexpected character (at character 1)
  <html>
  ^
  
  dart:convert                                     JsonCodec.decode
  package:openfoodfacts/openfoodfacts.dart 369:14  OpenFoodAPIClient.getIngredientSpellingCorrection
  
00:31 +29 -3: Some tests failed. 
```
The first two errors are probably just a consequence of changes in the database. So probably the tests should be rewritten in a way to avoid this issue. The last error ("get robotoff ingredient..") is different because the error only happens occasionally (it does not produce every time). 

So in short the problem might not be the SDK itself but rather the tests. Nevertheless, this definitely complicates enabling CI (#3).

**Comment by PrimaelQuemerais** [comment]
Closing old issue, not relevant anymore
